engine/table/src/main/java/io/deephaven/engine/table/impl/hierarchical/RollupTableImpl.java

```diff
@@ -80,6 +80,14 @@ public class RollupTableImpl extends HierarchicalTableImpl<RollupTable, RollupTa
     private final AggregationRowLookup[] levelRowLookups;
     private final ColumnSource<Table>[] levelNodeTableSources;
 
+    /**
+     * When {@link #withFilter(Filter)} is called, we apply the filters to the lowest level, and re-aggregate. We need
+     * them to keep it around, so that when we rebase to a new table we can re-apply them and preserve the semantics of
+     * our chain of rollup operations.
+     */
+    @Nullable
+    private final WhereFilter[] lowestRollupKeyFilters;
```

Comment:
This surprised me, but I guess it's right. You'll have to add that as a note to the JavaDoc suggestion.
I think "lowest" just makes this more confusing to read.